web/test/spec/filterbar/numeric-range-directive.spec.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+describe('driver.filterbar: Numeric Range', function () {
```

Comment:
I'd like to see a test here that checks the validation logic. Also, I noticed this directive doesn't have an associated controller: it may be easier to test this if the controller is broken out.